src/components/menuItemPropTypes.js

```diff
@@ -67,9 +67,9 @@ const propTypes = {
     /** A fallback avatar icon to display when there is an error on loading avatar from remote URL. */
     fallbackIcon: PropTypes.func,
 
-    /** If we need to show a brick road indicator or not. For now only value allowed is `error`,
-     * but we will add `success` later for manual requests */
-    brickRoadIndicator: PropTypes.oneOf(['error']),
+    /** If we need to show a brick road indicator or not. For now only value allowed is 'error' or '',
+     * but we will add 'success' later for manual requests */
+    brickRoadIndicator: PropTypes.oneOf([CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR, '']),
```

Comment:
The empty string is the empty value so that functions that get the brick road indicator property can always return a string.